CONTRIBUTING.md

```diff
@@ -168,6 +170,29 @@ Please remember to:
 Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels ((enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/aws/aws-step-functions-data-science-sdk-python/labels/help%20wanted) issues is a great place to start.
 
 
+## Setting Up Your Development Environment
+
+Setting up your IDE for debugging your tests locally will save you a lot of time.
+You might be able to `Run` and `Debug` the tests directly in your IDE with your default settings, but if it's not the case,
+follow the steps described in this section.
+
+### PyCharm
+1. Set your Default test runner to `pytest` in _Preferences → Tools → Python Integrated Tools_
+1. Go to _Preferences → Build, Execution, Deployment → Python Debugger_ and set the options with following values:
+   
+   | Option                                                      | Value                 |
+   |------------------------------------------------------------:|:----------------------|
+   | Attach subprocess automatically while debugging             | `Enabled`             |
+   | Collect run-time types information for code insight         | `Enabled`             |
+   | Gevent compatible                                           | `Disabled`            |
+   | Drop into debugger on failed tests                          | `Enabled`             |
+   | PyQt compatible                                             | `Auto`                |
+   | For Attach to Process show processes with names containing  | `python`              |
+1. Right click on a test or test file and select `Run/Debug`
+   
+    _Note: Can also be done by clicking on green arrow next to test definition_
```

Comment:
should there be a link in the `Running Unit Tests` section that points towards running them in the IDE? 
or should this be a part of the `Running Unit Tests` section as a sub-topic?

Reply:
+1. This is relevant for integ and unit tests

Reply:
Agreed, this belongs in the integ and unit test section.
Will include the changes in the next commit